osquery/tables/system/linux/md_tables.cpp

```diff
@@ -124,6 +124,10 @@ std::string MD::getPathByDevName(const std::string& name) {
   walkUdevDevices("block", [&](udev_device* const& device) {
     const char* devName = udev_device_get_property_value(device, "DEVNAME");
 
+    if (name.length() > strlen(devName)) {
```

Comment:
Maybe this is too much optimization, but `strlen(devName)` walks the whole string, then we do the same thing again on line 132. If we calc `auto dev_name_length = strlen(devName);` we'll only walk once. :P